### ISSUE TITLE
don't use style-loader and css-loader - unblock https://github.com/rebelstackio/yakchat/issues/20

### ISSUE THREAD

**Issue by ReinsBrain** [rebelstackio/yakchat#45]
unblock https://github.com/rebelstackio/yakchat/issues/20
- [ ] remove style-loader and css-loader
- [ ]  use parcel and kill another ticket at same time
